Subject: Remove fpu_control.h include
Upstream: No
Author: Simon Frankenberger <simon-alpine@fraho.eu>

The header is not present with musl and including it results in build error.
It's not needed anyways.

--- old/src/hotspot/os_cpu/linux_aarch64/os_linux_aarch64.cpp
+++ new/src/hotspot/os_cpu/linux_aarch64/os_linux_aarch64.cpp
@@ -75,7 +75,6 @@
 # include <pwd.h>
 # include <poll.h>
 # include <ucontext.h>
-# include <fpu_control.h>
 
 #define REG_FP 29
 #define REG_LR 30
